quantmsio/commands/utils/project.py

```
@@ -0,0 +1,86 @@
+    project_handler.populate_from_sdrf(str(sdrf_file))
+    project_handler.add_quantms_version(quantmsio_version=QUANTMSIO_VERSION)
+    project_handler.add_software_provider(
+        sortware_name=software_name, sortware_version=software_version
+    )
+    project_path = str(output_folder / "project.json")
+    project_handler.add_sdrf_file(
+        sdrf_file_path=str(sdrf_file),
+        output_folder=str(output_folder),
+        delete_existing=delete_existing,
+    )
+    project_handler.save_updated_project_info(output_file_name=project_path)
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'output_file_name' in method call

(E1123)